File: Python Code/Daily_Flash_Code_0/Week 2/22Jan/08-DailyFlash_Solutions/08-DailyFlash_Solutions/22_Jan_Solutions_Three/Python/p4.py
'''
Program 4: Write a Program to simulate simple calculator. Accept two integers
from user and sign of operation to perform.
Input:
Enter First Number: 10
Enter Second Number: 20
Enter the sign of Operation: +
Output: Addition of 10 & 20 is 30.
'''

var1=int(input("enter first integer "))
var2=int(input("enter second integer "))
sign=input("enter sign of operation")

if(sign == "+"):
	add = var1+var2
	print("addition is",add)

elif(sign == "-"):
	sub = var1-var2
	print("subtraction is",sub)

elif(sign == "*"):
	mul =var1*var2
	print("multiplication is",mul)

elif(sign == "/"):
	div= var1/var2
	print("division is",div)

else:
	print("invalid input")

File: Python Code/Daily_Flash_Code_0/Week 2/22Jan/08-DailyFlash_Solutions/08-DailyFlash_Solutions/22_Jan_Solutions_Three/C/p4.c
/*
Program 4: Write a Program to simulate simple calculator. Accept two integers
from user and sign of operation to perform.
Input:
Enter First Number: 10
Enter Second Number: 20
Enter the sign of Operation: +
Output: Addition of 10 & 20 is 30.
*/

#include<stdio.h>


int main(){
	int var1,var2,sub=0,add=0,mul=0;
	float div;
	char sign;


	printf("Enter two integers : \n");
	scanf("%d",&var1);
	scanf("%d",&var2);

	printf("Enter sign of operation : \n");
	scanf(" %c",&sign);

	switch(sign){
		case '+' :add=var1+var2;
			printf("addition of %d & %d is %d\n",var1,var2,add);
			  break;
		
		case '-':sub=var1-var2;
			printf("subtraction of %d & %d is %d\n",var1,var2,sub);
			 break;
		
		case '*':mul=var1*var2;
			printf("multiplication of %d & %d is %d\n",var1,var2,mul);
			 break;

		case '/':div=var1/var2;
			printf("division of %d & %d is %f\n",var1,var2,div);
			break;

		default:printf("Invalid input\n");

	}

	return 0;
}


